container/webui/docker-compose.yaml

```diff
@@ -136,7 +135,12 @@ services:
     volumes:
       - ./workdir/db:/var/lib/postgresql/data:Z
     healthcheck:
-      test: ["CMD", "sh", "-c", "echo 'select * from api_keys;' | psql -U openqa -v 'ON_ERROR_STOP=1' openqa"]
+      test:
+        - CMD-SHELL
+        - >-
+          pg_isready -U openqa -d openqa
```

Comment:
shouldnt use a retry here?

Reply:
No, why? This is a healthcheck probe: https://docs.docker.com/reference/dockerfile/#healthcheck, the retry is "built-in".